Testing/SystemTests/tests/qt/PythonInterfacesStartupTest.py

```diff
@@ -0,0 +1,71 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import os
+import systemtesting
+
+from mantid.kernel import ConfigService
+from mantidqt.utils.qt.testing import get_application
+
+from qtpy.QtCore import QCoreApplication, QSettings
+
+
+# Frequency_Domain_Analysis_Old.py  -  Excluded because is being deleted in Mantid Version 6.0
+# Frequency_Domain_Analysis.py      -  Excluded because it is causing a crash
+EXCLUDED_SCRIPTS = ["Frequency_Domain_Analysis_Old.py", "Frequency_Domain_Analysis.py"]
```

Comment:
Just a quick question about `Frequency_Domain_Analysis`. I remember we discussed it but was it not the Qt4/5 issue or was it something else?

Reply:
For this interface it appears to be something else. I have tried to debug it but I couldn't figure out what was going on.

I think someone with more experience or better knowledge of the Muon area might be able to work it out. Can I create a separate maintenance issue for this to be fixed and incorporated into this test?

Reply:
Yes please do create a separate issue. We need the tests to be reliable indicators of if something is wrong so it would be good to figure this out.